src/core/js/adapt.js

```diff
@@ -74,19 +74,25 @@ define([
     Adapt.componentStore = {};
     Adapt.mappedIds = {};
 
-    Adapt.initialize = _.once(function() {
+    Adapt.initialize = function() {
 
         //wait until no more completion checking 
         Adapt.deferUntilCompletionChecked(function() {
 
             //start adapt in a full restored state
             Adapt.trigger('adapt:start');
-            Backbone.history.start();
+
+            if (!Backbone.History.started) {
+                Backbone.history.start();
+            }
+
+            Adapt.set("_isStarted", true);
```

Comment:
This unrelated line has crept in from https://github.com/adaptlearning/adapt_framework/pull/1524.